src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php

```diff
@@ -21,11 +21,14 @@
  */
 class Query extends AbstractQuery
 {
+    // As of PHP 7.2, we can use LDAP_CONTROL_PAGEDRESULTS instead of this
```

Comment:
Isn't an underscore missing in `PAGEDRESULTS`?

Reply:
Nope, LDAP_CONTROL_PAGEDRESULTS is the name of the OID constant that libldap uses.